test/test_utils/__init__.py

```diff
@@ -191,7 +191,7 @@ def get_ami_id_ssm(region_name, parameter_path):
 
 SAGEMAKER_EXECUTION_REGIONS = ["us-west-2", "us-east-1", "eu-west-1"]
 # Before SM GA with Trn1, they support launch of ml.trn1 instance only in us-east-1. After SM GA this can be removed
-SAGEMAKER_NEURON_EXECUTION_REGIONS = ["us-east-1"]
+SAGEMAKER_NEURON_EXECUTION_REGIONS = ["us-west-2"]
```

Comment:
Approved for this file. 
Note: Make sure to revert/modify the usage for this variable if this setting is not applicable for other neuron or neuronx images that requires instance type in us-east-1